angr/engines/concrete.py

```diff
@@ -37,23 +37,26 @@ def __init__(self, project):
 
         self.segment_registers_already_init = False
 
-    def _check(self, state, *args, **kwargs):
+    def __check(self, state, *args, **kwargs):
         return True
 
-    def _process(self, new_state, successors, *args, ** kwargs):
+    def process_successors(self, successors, *args, ** kwargs):
+        new_state = self.state
         # setup the concrete process and resume the execution
-        self.to_engine(new_state, kwargs['extra_stop_points'], kwargs['concretize'], kwargs['timeout'])
+        self.to_engine(new_state, kwargs['extra_stop_points'], kwargs['memory_concretize'],
+                       kwargs['register_concretize'], kwargs['timeout'])
```

Comment:
You can just make these actual keyword arguments instead of extracting them from the kwargs dict. also, what's with the space between `**` and `kwargs`?